_docs_use-cases/use-case-transport-fill-device-onboarding-token-with-new-device.md

```diff
@@ -0,0 +1,58 @@
+---
+# Start automatic generation
+permalink: use-case-transport-fill-device-onboarding-token-with-new-device
+published: true
+title: "Fill Device onboarding Token with new Device"
+type: use-case
+toc: true
+sidebar:
+  - title: "Integrate enmeshed"
+    nav: "docs_integrate"
+properties:
+  - id: RD10
+  - component: Runtime
+  - layer: Transport
+  - facade: DevicesFacade
+  - function: fillDeviceOnboardingTokenWithNewDevice
+  - description: Fill Device onboarding Token with new Device
+  - feature category: Multi-device
+  - tech category: Devices
+  - status: DONE
+  - documentation status: DONE
+  - comments:
+  - actor: Identity
+  - trigger: Runtime
+  - precondition:
+  - result:
+  - priority: n/a
+  - complexity: n/a
+  - size: n/a
+  - created_at:
+  - changed_at:
+  - api_route_regex:
+  - published: default
+  - link: use-case-transport-fill-device-onboarding-token-with-new-device
+require:
+required_by:
+# End automatic generation
+---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use case fills the Device onboarding [Token]({% link _docs_integrate/data-model-overview.md %}#token) with a new Device.
```

Comment:
hm, that sounds wrong.. I'd say: creates a new device and fills the given empty token with the onboarding information.